README.md

```diff
@@ -155,6 +156,70 @@ StatusCode = {
 };
 ```
 
+## Support of IndexedDB
+This library provides the support of storing private keys in the browser [indexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API).
+The function can be enabled by setting `scalar.dl.client.private_key_indexeddb_enabled` to `true` in the client properties.
+
+As indexedDB function is enabled, depending on whether a private key is specified in the properties,
+the construction stores the private key in the indexedDB or reads the private key from the indexedDB.
```

Comment:
```suggestion
the construction will store the private key in indexedDB or read the private key from indexedDB.
```
In this sentence, I'm not sure what "construction" means, maybe it's not the right word?

Reply:
Ah, it means to execute constructor XD